setup.cfg

```diff
@@ -1,13 +1,23 @@
 [flake8]
 # D103 Missing docstring in public function
-ignore = D103
+# WPS336 Found explicit string concatenation
+# I001 isort found an import in the wrong position
+# I003 isort expected 1 blank line in imports, found 0
+# WPS348 Found a line that starts with a dot
+ignore = D103, WPS336, I001, I003, WPS348
 
 per-file-ignores =
 # S101 Use of assert detected.
-    tests/*: S101
+# D100 Missing docstring in public module
+# WPS432 Found magic number
+    tests/*: S101, D100, WPS432
+    src/*/*: D100
 
 exclude = docs, .venv, .git
 
-
 [isort]
 profile = wemake
+force_grid_wrap = 2
```

Comment:
What does this do? I read docs but still a little confused

Reply:
When you do:

```
from package import fun1, fun2, fun3
```

It will automatically wrap it into multiple lines if you import 2 or more functions:

```
from package import (
  fun1,
  fun2,
  fun3,
)
```